cudaextender/src/ungapped_xdrop_kernels.cu

```diff
@@ -327,8 +359,36 @@ __global__ void find_high_scoring_segment_pairs(const int8_t* __restrict__ d_tar
 #pragma unroll
             for (int32_t offset = 1; offset < warp_size; offset = offset << 1)
```

Comment:
@ahehn-nv Since we're using bitwise shift operator here, would it be better to keep `offset` unsigned, though it might never reach the sign bit?

Reply:
I understand your concern, but I'd keep `offset` signed, since it still represents a number, not a bit pattern or something. Shifting too far would be similar to an integer overflow by regular multiplication or addition.